add `make install` target

The usual workflow for installation is:

```
cmake
make
[sudo] make install
```

At the moment, w2rap-contigger does not provide an automated way to install the built executables:

```console
$ make install
make: *** No rule to make target `install'.  Stop.
```

---

PR incoming, just give me a sec.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -409,3 +409,8 @@ if (DEFINED MALLOC_LIBRARY)
     endif()
 endif()
 
+# -----------------------------------------------------------------------------
+# installation
+# -----------------------------------------------------------------------------
+
+install(TARGETS w2rap-contigger hbv2gfa DESTINATION bin)

